Makefile

```diff
@@ -15,7 +15,7 @@ test:
 	rm -rf project_new/
 	pip install .
 	create-aio-app project_new
-	cd project_new/ && docker-compose up project_new_app
+	cd project_new/ && docker-compose up test  && docker-compose stop
```

Comment:
if test would fail docker will not stop build

Reply:
Good point, travis will clean that up anyway, but probably there is magic in docker-compose to handle issue gracefully.